agreements/project-contribution/terms-and-conditions.md

```diff
@@ -1,4 +1,293 @@
+
+The Project must work with the Sponsor to ensure financial records are accurate, including management/reconciliation of funds received or expenses paid using mechanisms other than the Platform.
+
+### Transparency and Reporting
+
+Projects must maintain transparency in their financial activities for funds managed under the Sponsor’s oversight and provide regular updates to the Sponsor.
+The Sponsor will compile an annual financial report for each Project, detailing income, expenses, and remaining balances for these Sponsor-managed funds.
+
+Projects will be asked to review and verify the financial summary provided by the Sponsor to ensure accuracy.
+Any discrepancies should be reported to the Sponsor within 30 days of receiving the report.
+
+Projects are encouraged to keep their expense tracking up to date on the Open Collective platform for real-time transparency.
+
+### Independent Financial Management
+
+Individuals, companies, or groups may raise and manage funds independently in support of their contributions to the Project.
+These funds are not part of the fiscal sponsorship arrangement and must remain separate from the Sponsor’s financial management systems.
+Those managing independent funds are solely responsible for compliance with all relevant laws and regulations.
+
+While alignment with the Sponsor’s policies is encouraged where possible, these independent activities must remain separate from the Project’s operations under fiscal sponsorship.
+
+## Continuity and Leadership Responsibilities
+
+Project leadership and operations naturally evolve due to changes in roles and responsibilities, or to unforeseen circumstances.
+
+### Succession Planning and Leadership Transition
+
+In this section, **“Successor”** refers to a qualified individual who assumes the role of Project Leader or Maintainer.
+As specified in the [Fiscal Sponsorship Agreement][fs-landing], a Successor must agree to the terms of that Agreement and become a Signatory.
+
+**Effort to Identify Successors:**
+Projects should establish and maintain a continuity plan that includes making a good faith effort to identify and prepare qualified Successors.
+
+**Resignation of Signatories:**
+Signatories may resign by providing written notice to the Sponsor.
+Resignation removes the Signatory from the agreement, but does not terminate the agreement or alter the Sponsor’s ownership of the Project and its assets.
+
+If a Project has only one Signatory and they wish to resign without a designated Successor, they must take all reasonable steps to enable the Sponsor to assume operational responsibility.
+This includes:
+
+- Documenting critical processes and procedures.
+- Transferring access to all relevant accounts, tools, and resources.
+- Replacing any personal or employer-provided resources with Sponsor-accessible alternatives.
```

Comment:
Should something about "transferring project-restricted funds to Sponsor" be included in this list?

Reply:
This is the succession/transition bucket (project leadership changes after agreements have been signed). All project-specific restricted funds are already managed by us.